src/www/css/app.css

```diff
@@ -1350,6 +1350,11 @@ video {
   transition-timing-function: cubic-bezier(0, 0, 0.2, 1);
 }
 
+.is-disabled {
```

Comment:
I'm guessing you have named this class `is-disabled` to differenciate from the `disabled` state? I like your thinking and think we can build on this. Would you be happy to rename to `inactive`? Allows, then, for potential wider uses! :)